### PR TITLE
Prepare for transport reassembler component

### PR DESCRIPTION
**Pull request by BenGardiner**

### CODE REVIEW

File: test_j1587_driver.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        if self.memo_fake_driver is None:
+            self.memo_fake_driver = FakeJ1708Driver()
+        return self.memo_fake_driver
+
+    def clear(self):
+        self.memo_fake_driver = None
+
+
+class J1587TestClass(unittest.TestCase):
+    def setUp(self):
```
Comment (haystack-ia) [inline test_j1587_driver.py:56]: bruh don't use camel case in method names

File: test_j1587_driver.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        self.fake_j1708_factory = FakeJ1708Factory()
+        set_j1708_driver_factory(self.fake_j1708_factory)
+        self.j1708_driver = self.fake_j1708_factory.make()
+
+    def tearDown(self):
```
Comment (haystack-ia) [inline test_j1587_driver.py:61]: ibid

**Review by haystack-ia — CHANGES_REQUESTED** [review]
Overall I think these changes are sensible and make the code more Professional Grade than it was previously.

I'd just ask that a couple method names be changed to snake_case rather than CamelCase for consistency.

File: test_j1587_driver.py (hunk 1 above)
Comment (BenGardiner) [reply test_j1587_driver.py:56]: lol I wish I could choose not too broseph. its the unittest.TestCase method override.

File: test_j1587_driver.py (hunk 2 above)
Comment (BenGardiner) [reply test_j1587_driver.py:61]: 😘

**Comment by BenGardiner** [comment]
Thanks for reviewing @haystack-ia !

**Review by haystack-ia — COMMENTED** [review]
strike that, was informed why the camelcase is there

**Pull request merged**